### DISCUSSION TITLE
官网在github种提供的脚本参数错误

### DISCUSSION THREAD

**Discussion by YSYAdvisable** [apache/doris#28040] [RFC]
![1701829734288](https://github.com/apache/doris/assets/82552426/24817071-b0c8-4ebe-828c-2008ca9cc92b)
![1701829799982](https://github.com/apache/doris/assets/82552426/3705de86-ba0b-4b08-b638-51a6c85237c6)
![1701829833529](https://github.com/apache/doris/assets/82552426/443946a7-5156-491e-ad6c-07e810cdd849)
![1701829867370](https://github.com/apache/doris/assets/82552426/7e4598b2-a111-40e0-94f5-094da0f9812f)

根本无此参数，这个参数写错名字了。

**Comment by LemonLiTree** [comment]
This method is no longer recommended. It is now recommended to use jdbc catalog.
